docker-compose.yml

```diff
@@ -11,12 +11,12 @@ services:
       - "7890:7890"
     profiles:
       - mocksls
-  db:
-    image: postgres
-    env_file:
-      - docker-compose/db-env-vars.env
-    ports:
-      - "5432:5432"
+  # db:
+  #   image: postgres
+  #   env_file:
+  #     - docker-compose/db-env-vars.env
+  #   ports:
+  #     - "5432:5432"
```

Comment:
@JFU-GIT Was doing some local testing and this file needed to be restored back to include the DB.  This might have been something you commented out while testing?

Reply:
OH, I commented out db to use the sqllite3, because I run into problem with docker - it stopped being able to resolve the 'db' service name (I don't know the cause yet).

I can restore it to postgres since may be that issue only happen on windows.